packages/upload/stories/upload.stories.tsx

```diff
@@ -77,7 +76,7 @@ export default {
   // ],
```

Comment:
`Components/Upload/Upload`

also, i'm not as familiar with this component but are the mocks still working when it's not in the story decorator? if it does we should remove this commented out code